### ISSUE TITLE
`dj.U()` with no argument doesn't work as expected

### ISSUE THREAD

**Issue by eywalker** [datajoint/datajoint-python#246]
Given a sample table such as the one below:

```python
@schema
class Test(dj.Computed):
    definition="""
    key   :   int     # key
    ---
    value   :   int     # value
    """
    contents = [(k, k) for k in range(10)]
```

The following query using `dj.U()` with no argument

```python
dj.U().aggregate(Test(), vmax='max(value)')
```

is expected to return the maximum value of the column `value` across all tuples in `Test`. However, the query results in the following error instead:

```python
InternalError: (1054, "Unknown column '' in 'group statement'")
```

Inspection of SQL gives:

```sql
SELECT max(value) as `vmax` FROM `dj_test`.`__test` GROUP  BY ``
```

**Comment by dimitri-yatsenko** [comment]
yes, the `GROUP BY` clause should be omitted when the grouping attributes are empty.

**Comment by eywalker** [comment]
Cool, thanks!

**Comment by dimitri-yatsenko** [comment]
fixed in #251.

**Comment by eywalker** [comment]
PR #251 merged and verified to fix this issue.
